### ISSUE TITLE
Chocolatey warns that no programs will be uninstalled, but uninstalls successfully anyways

### ISSUE THREAD

**Issue by steviecoaster** [chocolatey/chocolatey-licensed-issues#67]
### What You Are Seeing?

Chocolatey is warning that multiple keys are returned for a package. It then states that no applications will be removed to prevent data loss, however the applications are in fact uninstalled

### What is Expected?

Chocolatey should halt the uninstallation based on the warning message

### How Did You Get This To Happen? (Steps to Reproduce)
`choco uninstall _packagename_`

### References

* [Internal Issue](https://gitlab.com/chocolatey/collaborators/choco-licensed/issues/267)

**Comment by ferventcoder** [comment]
This is typically produced by `chocolateyUninstall.ps1` using the `Get-UninstallRegistryKey` function, followed by AutoUninstaller doing the uninstall.

So you have two different items here. AutoUninstaller is more deterministic and it is able to perform the uninstallation of software that a package has installed on uninstall without a need for an uninstall script. If you have both and one of them is not quite right, that's where you see this.

In short, this is a package issue.

**Comment by ferventcoder** [comment]
For future me: Can't seem to add a label to this one.

**Comment by ferventcoder** [comment]
https://twitter.com/ferventcoder/status/1110634937970032640

**Comment by ferventcoder** [comment]
So I can add the label "Enhancement" and remove it.
